WorldModel/WorldModel/Functions/DateTimeFunctions.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace TextAdventures.Quest.Functions
+{
+    public static class DateTimeFunctions
+    {
+        public static string CurrentDate()
+        {
+            return CurrentDate(0);
+        }
+        public static string CurrentDate(int offset)
+        {
+            DateTime localDate = DateTime.Now;
+            double offsetD= Convert.ToDouble(offset);
+            localDate = localDate.AddHours(offsetD);
```

Comment:
You don't need to do the conversion here, you can just pass the integer `offset` straight into `AddHours`.

Reply:
That's right. I tried to use optional parameter like int? offset=null
There it said its not possible to convert it to double